urdf2webots/writeProto.py

```diff
@@ -224,6 +245,10 @@ def URDFBoundingObject(proto, link, level, boxCollision):
         proto.write(level * indent + '}\n')
 
 
+def defName(name):
+    return name.replace(' ', '_').replace('.', '_')
```

Comment:
The name of this function lets think that it is a getter.
I would rather use an action in the name, e.g. `cleanDefName(name)`